README.md

```diff
@@ -669,6 +669,41 @@ Here is an example of using it with the `screenWithOneInput` fragment:
 
 Note that passing `required=true` here will append the red `(required)` to the label.
 
+#### Special situation for requiring address fields
+
+Sometimes you may want to apply custom validations to address fields through actions that effectively
```

Comment:
It might be good to add a pointer to the action for custom validations, so that they fully understand what this is referring to.   Maybe capitalize "Actions" and have it be a link to "https://github.com/codeforamerica/form-flow?tab=readme-ov-file#actions"?   Same for the date fields section below.

Reply:
Done